common/autoboot.c

```diff
@@ -457,8 +457,9 @@ const char *bootdelay_process(void)
 		disconnect_from_pc();
 		printf("Boot from USB for mfgtools\n");
 		bootdelay = 0;
-		env_set_default("Use default environment for \
-				 mfgtools\n", 0);
+		// Disabled by Polytech - at this point it wipes some important enviroment variables such as mac addresses
+		// env_set_default("Use default environment for \
```

Comment:
I thought a long time trying to merely just get rid of having `bootcmd_mfg` set in the enviroment at all but there is several layers of logic in there that prevented booting the OS in the end. This works and as we in the production tool initially ensure the enviroment is reset to default it should be just the same.